src/Raven.Client/Raven.Client.csproj

```diff
@@ -44,16 +44,16 @@
   </ItemGroup>
   <ItemGroup>
 	<PackageReference Include="Lambda2Js.Signed" Version="3.1.0" />
-    <PackageReference Include="Microsoft.CSharp" Version="4.4.1" />
-    <PackageReference Include="System.Buffers" Version="4.4.0" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.5.0-preview2-26406-04" />
+    <PackageReference Include="System.Buffers" Version="4.5.0-preview2-26406-04" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
     <PackageReference Include="Raven.CodeAnalysis" Version="1.0.10">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.4.0" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.0-preview2-26406-04" />
```

Comment:
I don't know, but it seems to me that standard 2.0 shouldn't use ```4.5.0-preview2```, probably we should separate those in different compilation targets.

Reply:
No, those are just updated versions of packages that we use, they are targetting more than just a latest .net core 2.1. Check dependencies of this package here for example: https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/4.5.0-preview2-26406-04

```
.NETCoreApp 2.0
.NETFramework 4.5
.NETStandard 1.0
.NETStandard 2.0
Portable Class Library (.NETFramework 4.5, Windows 8.0, WindowsPhone 8.0, WindowsPhoneApp 8.1)
Windows 8.0
WindowsPhone 8.0
WindowsPhoneApp 8.1
```

Reply:
Great!!!... some of those may not be available though like ```System.Runtime.Intrinsics``` so we need to be very careful

Reply:
Indeed

Reply:
This is why I've wrote that maybe internall projects like Voron might end up with 'netcoreapp2.1' target instead of 'netstandard2.0' one. Note that I've added already 'netcoreapp2.1' to Sparrow. You can use `#if NETCOREAPP` constant for conditional compilation